File: src/java/WebNinja/src/main/java/webninja/AssignedIssuesReport.java
package webninja;

import java.util.List;

public class AssignedIssuesReport {
    private final List<Issue> issues;

    public AssignedIssuesReport(List<Issue> issues) {
        this.issues = issues;
    }

    public List<Issue> geIssues() {
        return issues;
    }

    public int getNumberOfIssues() {
        return issues.size();
    }
}

File: src/dotnet/WebNinja/WebNinja/webninja/AssignedIssuesReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WebNinja.webninja
{
    public class AssignedIssuesReport
    {
        public AssignedIssuesReport(IList<Issue> issues)
        {
            Issues = issues;
        }

        public IList<Issue> Issues { get; private set; }

        public int NumberOfIssues
        {
            get { return Issues == null ? 0 : Issues.Count; }
        }
    }
}
